src/plugins/intel_gpu/src/graph/impls/ocl/register.cpp

```diff
@@ -15,6 +15,7 @@ void register_implementations() {
     REGISTER_OCL(border);
     REGISTER_OCL(broadcast);
     REGISTER_OCL(bucketize);
+    REGISTER_OCL(col_to_im);
```

Comment:
Did you use the name `col_to_im` instead of `col2im` on purpose? I think it would be better to use the original name `col2im` to avoid confusion. (for other places too)